Psychtoolbox/PsychHardware/EyelinkToolbox/EyelinkBasic/EyelinkBasicLegacy/EyelinkLegacyDoTrackerSetup.m

```diff
@@ -0,0 +1,4 @@
+function result=EyelinkLegacyDoTrackerSetup(el, sendkey)
```

Comment:
Where's the legacy code or call back into the new tracker setup?

Reply:
resolved in 33984c0ef6fc9d2a3dc40f30ab9710ad031d9938